src/bundler.ts

```diff
@@ -282,6 +282,12 @@ export class Bundler {
     for (const bundle of bundles) {
       for (const exclude of this.excludes) {
         bundle.files.delete(exclude);
+        const excludeAsFolder = exclude.endsWith('/') ? exclude : exclude + '/';
```

Comment:
Doesn't seem like this would work under Windows -- should you use `path.sep` instead of `/`? Also, do we need to check for any special path operators? Maybe call `path.normalize` first?